### PR TITLE
Fix logic check, small improvements.

### PR DESCRIPTION
**Pull request by stefangordon**

### CODE REVIEW

File: ip-geo.html [hunk 1]
```diff
@@ -1,75 +1,93 @@
...
 
-            // use the value of enableBot to determine whether or not to display the healthbot
-            if (enableBot) {
-                healthBotEnabled.textContent = "ENABLED";
-            } else {
-                healthBotEnabled.textContent = "DISABLED";
-                var disabledMessage = document.getElementById("disabled_message");
-                disabledMessage.innerHTML = "<p>" + disabledMessageText + "</p>";
-            }
+        /* Now we take action based on enableBot result
```
Comment (aluong) [inline ip-geo.html]: Should we just move this `action` of updating some html element outside and have this function simply return true/false?

File: ip-geo.html [hunk 2]
```diff
@@ -1,75 +1,93 @@
...
-            region.textContent = json.region;
+    /* This function uses IP location information to determine if the health bot should be enabled */        
+    function determineBotEnabled(json) {        
+        // message to display if the client country and state is disabled
+        /* MODIFY THIS TO CHANGE BOT DISABLED DISPLAY TEXT */
+        var disabledMessageText = "The health bot is not currently available in your area.";
 
-            // message to display if the client country and state is disabled
-            /* MODIFY THIS TO CHANGE BOT DISABLED DISPLAY TEXT */
-            var disabledMessageText = "The health bot is not currently available in your area. THIS MESSAGE ONLY SHOWS IF DISABLED BY COUNTRY AND STATE!!!";
```
Comment (aluong) [inline ip-geo.html:36]: Thank you for stop yelling. =)

File: ip-geo.html (hunk 1 above)
Comment (stefangordon) [reply ip-geo.html]: Probably.  :) 

Lets take this one since it works and improves, and then keep iterating.

**Review by aluong — APPROVED** [review]

**Comment by aaron-schnieder** [comment]
LGTM

**Pull request merged**